modules/postman-templating/src/template-client.ts

```diff
@@ -102,47 +67,43 @@ export class TemplateClient {
             )
           }
 
-          // add key regardless, note that this is also returned in lowercase
           variables.add(key)
 
           // if no params continue with the loop
-          if (!params) return
+          if (!params) continue
 
-          // if params provided == attempt to carry out templating
           if (dict[key]) {
             const templated = dict[key]
             tokens.push(templated)
-            return
+          } else {
+            // recipient key must have param
+            if (key === 'recipient') {
```

Comment:
The previous code pattern was cleaner.  Return early, dont nest ifs if possible. 
https://blog.codinghorror.com/flattening-arrow-code/

Reply:
Ok yeah I can fix this, I just personally thought having an explicit `else` was clearer.